models/Cell.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+// Cell : Represent json model Data
```

Comment:
For each comment



No need to use : inside comments, this will made the godoc ugly

Make complete comments (see how other Go packages (Mux, Cors, Websocket...) did)